src/js/actions/transform.js

```diff
@@ -573,7 +585,8 @@ define(function (require, exports) {
 
             payload.size = newSize;
             
-            dispatchPromise = this.dispatchAsync(events.document.history.optimistic.RESIZE_DOCUMENT, payload);
+            dispatchPromise = this.dispatchAsync(events.document.history.RESIZE_DOCUMENT, payload);
+            // TODO is this missing historyStateInfo?
```

Comment:
I'm interested to know, but I'm also now going back through these comments to unspool them myself.  This was primarily a note-to-self.